### ISSUE TITLE
Statement of Need

### ISSUE THREAD

**Issue by pschloss** [zdelrosario/data-science-curriculum#25]
The statement of need could be more clear in the README and in the paper. There are active learning materials out there - e.g., The Carpentries, Codeacademy. What is the particular niche that your materials are filling?

I'm not sure that the "Notably, if teachers held ..." sentence is constructive. Surely the published books, blog posts, and videos have instructional value even if they aren't examples of active learning materials?

**Comment by firasm** [comment]
Adding my comments to this issue so the reviewer feedback is consolidated in one place:

## Statement of need 

- [ ] Nice, if a bit concise. I would suggest though that if you're going to name resources, it would be helpful to a bit more selective. It's not so useful to point out "Medium Blog posts" (but statquest is a good specific reference). Others might be data8, learnr project, the "hadley books" (https://r4ds.had.co.nz), data science: a first introduction (https://ubc-dsci.github.io/introduction-to-datascience/). Not necessarily advocating including these, these are just some that I know.

- [ ] Glad to see the Freeman reference advocating for active learning in the classroom. Might be worthwhile to make the point that good resources are essential for self-study outside of lecture time, but in the classroom active learning shows substantial learning benefits.

- [ ] May be worth discussing the importance of open education resources (OER) to making learning more equitable.

**Comment by zdelrosario** [comment]
@pschloss good point about articulating a specific need; I have updated the paper to emphasize that the present content has a stronger emphasis on engineering content than is typical in a data science curriculum. Is is appropriate (and possible) to modify the paper title to reflect this?

I also agree that my prior use of the Freeman et al. reference was unproductive; I have rephrased.

@firasm I have added some of the references you suggested; highlighted the potential of passive resources for self-study (and reference), and added your point about the equity benefits of OER.
